include/ebpf_structs.h

```diff
@@ -223,6 +223,30 @@ enum bpf_prog_type
      */
     BPF_PROG_TYPE_SOCK_OPS,
 
+    /** @brief Program type for handling netevents.
+     * ntosebpfext repo has the implementation for this program type.
```

Comment:
I don't think "ntosebpfext repo" is suffciently unambigous.
```suggestion
     * The github microsoft/ntosebpfext repo has the implementation for this program type.
```